snippets/maxN.md

```diff
@@ -0,0 +1,19 @@
+### maxN
+
+Returns the `n` maximum elements from the provided array. If `n` is greater than or equal to the provided array's length than return the original array(sorted in descending order).
+
+Sort's the array's shallow copy in descending order and returns the first n elements
+
+Skip the second argument to get a single element(in the form of a array)
+``` js
+const maxN = (arr,n = 1) => {
+  let rra = [...arr].sort((a,b) => b-a)
+  return rra.splice(0,n)
+}
```

Comment:
This can be done on a single line

```js
const maxN = (arr, n = 1) => [...arr].sort((a, b) => b - a).slice(0, n);
```

Reply:
@atomiks Ok I shall update the snippets 👍